### DISCUSSION TITLE
Updating Documentation Require Incrementing Patch Version?

### DISCUSSION THREAD

**Discussion by bpoole6** [argoproj/argo-helm#2961] [Q&A] [answered]
I want to update some of the documentation. Does this require that I also increment the patch version as well?

**Comment by yu-croco** [comment, marked as answer]
It depends which dir you are planning to update but usually you need to bump version.
For more info, please refer to https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md
